### DISCUSSION TITLE
Issue with alpine install on sage

### DISCUSSION THREAD

**Discussion by RobDobsonNC** [alpinejs/alpine#3440] [7. Bugs (archived)]
Hello - hope you can help. I've got a fresh install of sage.

additional modules of Log1X ACF Composer, and Sage Directives - everything works fine. 

I then go into themes/mytheme and run

`npm install alpinejs`

Which then stops sage compiling, giving me an internal error on` sage build.` :

`This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile`

I've run yarn install, and every other things I can think of, but still no luck. I've tried four times on different installs and still no luck. Does anyone have any ideas of how I can fix?

**Comment by ekwoka** [comment]
I'd love to help. What is sage?

**Reply by RobDobsonNC** [reply]
Apologies! https://roots.io/sage/docs/installation/

**Reply by ekwoka** [reply]
oh...wordpress.

I'd suspect it has something to do with how the system handles the lockfile. If it is containerizing things, it may not be copying over the updated lockfile when you go to run it.
